docs/en_US/Compressor/Framework.md

```diff
@@ -1,6 +1,16 @@
-# Design Doc
+# Customize A New Compression Algorithm
 
-## Overview
+```eval_rst
+.. contents::
+```
+
+To simplify writing a new compression algorithm, we design programming interfaces which are simple but flexible enough. There are interfaces for pruning and quantization respectively. Below, we first demonstrate how to customize a new pruning algorithm and then demonstrate how to customize a new quantization algorithm.
+
+## Customize a new pruning algorithm
+
+To better demonstrate how to customize a new pruning algorithm, it is necessary for users to first understand the framework for supporting various pruning algorithms in NNI.
+
+### Framework overview for pruning algorithms
```

Comment:
The title doesn't help to understand purpose of its content. The content should like a step by step guidance to create user's puring algorithm.

Reply:
also leave it to @chicm-ms :)